packages/react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -961,12 +961,12 @@ DatePicker.propTypes = {
   /**
    * The maximum date that a user can pick to.
    */
-  maxDate: PropTypes.string,
+  maxDate: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
Was throwing a warning since we use a number in the `.mdx` examples. It works with either a number or a string.